code/game/machinery/Sleeper.dm

```diff
@@ -79,7 +79,8 @@
 /obj/machinery/sleeper/close_machine(mob/user)
 	if((isnull(user) || istype(user)) && state_open && !panel_open)
 		..(user)
-		if(occupant && occupant.stat != DEAD)
+		var/mob/living/mob_occupant = occupant
+		if(mob_occupant && mob_occupant.stat != DEAD)

```

Comment:
so Remie, should I do type checking constantly, or just assume that we're not badminned?